### DISCUSSION TITLE
Does the code on "updates" branch work?

### DISCUSSION THREAD

**Discussion by kingkongwrdsv** [N-John/mikrotik_hermes#4] [Q&A] [answered]
From the code, there seems to be a separate branch (updates) which when I run seems to have issues, how do I run the code?

**Comment by N-John** [comment, marked as answer]
The code on "updates" gets pulled and pushed regularly. It holds unstable part of the program which is still being modified or is under testing. We recommend you use the code from the "main" branch.
